modules/account-lib/test/unit/coin/dot/transaction.ts

```diff
@@ -54,6 +54,14 @@ describe('Dot Transaction', () => {
       tx.sender(DotResources.accounts.account2.address);
       should.deepEqual(tx.canSign({ key: DotResources.accounts.account2.secretKey }), true);
     });
+
+    it('can generate valid txHash from signed transaction', () => {
+      const signedTx = DotResources.rawTx.transfer.westendSigned2;
+      const txHash = Utils.default.getTxHash(signedTx);
+      const expectedHash = '0x252e9b53c1d068c275ef4c9b5afcffb2df42859203be1305d148c0c1441a5b20';
+
+      expectedHash.should.equal(txHash);
```

Comment:
nit: the assertion should be the other way around (otherwise the assertion error message doesn't make much sense)
```suggestion
      txHash.should.equal(expectedHash);
```